verilator-config.cmake.in

```diff
@@ -261,11 +244,13 @@ function(verilate TARGET)
 
   if (${VERILATE_PREFIX}_TRACE_VCD)
     # If any verilate() call specifies TRACE, define VM_TRACE in the final build
+    set_property(TARGET ${TARGET} PROPERTY VERILATOR_TRACE ON)
     set_property(TARGET ${TARGET} PROPERTY VERILATOR_TRACE_VCD ON)
   endif()
 
   if (${VERILATE_PREFIX}_TRACE_FST)
     # If any verilate() call specifies TRACE_FST, define VM_TRACE_FST in the final build
+    set_property(TARGET ${TARGET} PROPERTY VERILATOR_TRACE ON)
```

Comment:
There is no `VERILATOR_TRACE` property defined.

Reply:
Properties don't need to be defined, it's just for documentation. And we don't actually want users to edit those properties. I'll add it anyway for completeness.